releasenotes/notes/release_fix_trace_of_products-9bb81dfc2724ed1d.yaml

```
@@ -0,0 +1,9 @@
+---
+fixes:
+  - |
+    Fixing a bug in the ``ProductFrame._trace_of_prod`` method. 
+    This method computes the trace of the product of two operators.
+    It requires at some point the value of ``Tr[(P_N)^2]`` where ``P_N`` is an ``N``-qubit Pauli string. 
+    The correct value of this trace is ``2^N`` but was previously set to ``2*N``. 
+    This release fixes this issue.
+    Note that this issue was only impacting ``MultiQubitFrame`` for ``num_qubit>=3`` or product of such frames.
```

I think it is fine if we keep it less detailed than that.
How about the following?
```suggestion
    Fixes a bug in the :meth:`.ProductFrame.analysis` method occurring for :class:`.MultiQubitFrame` instances
    with ``num_qubits>2``. This bug affected the outcome of other methods such as
    :meth:`.BasePOVM.get_prob` and :meth:`.BaseDual.get_omegas`.
```

---

Okay sounds good 👍